R/fit_SE.R

```diff
@@ -100,17 +101,27 @@ fit_FUN <- function(x,
       x <- x[x[[conc]] != 0, ]
     }
     
-    fit_df <- S4Vectors::DataFrame(gDRutils::fit_curves(
-      data.table::as.data.table(x),
-      series_identifiers = nested_identifiers,
-      e_0 = 1,
-      GR_0 = 1,
-      n_point_cutoff = n_point_cutoff,
-      range_conc = range_conc,
-      force_fit = force_fit,
-      pcutoff = pcutoff,
-      cap = cap,
-      normalization_type = curve_type))
+    fit_df <- S4Vectors::DataFrame(
+      gDRutils::fit_curves(
+        data.table::as.data.table(x),
+        series_identifiers = nested_identifiers,
+        e_0 = 1,
+        GR_0 = 1,
+        n_point_cutoff = n_point_cutoff,
+        range_conc = range_conc,
+        force_fit = force_fit,
+        pcutoff = pcutoff,
+        cap = cap,
+        normalization_type = curve_type)
+    )
+    
+    if (is.null(fit_df$fit_source)) {
+      rownames(fit_df) <- fit[["normalization_type"]]
+    } else {
```

Comment:
If these changes are required for working other PRs all the datasets should be reprocessed :/

Reply:
Ok, with [this fix in gDRutils](https://github.com/gdrplatform/gDRutils/pull/89/files#diff-769715cb0fa3d93b007073d337903aaf1348828ffecf761270e5d58073942982R125-R127), this PR will be closed. So, reprocessing of all the datasets will not be necessary.